README.md

```diff
@@ -1,2 +1,68 @@
-# musicartists
-An app to see the top music artists and display their information
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+### TITLE
+
+> ### HYRULE COMPENDIUM
+
+## Description
+
+"Hyrule Compendium" is a web application where you can discover your favorite elements of the 'Breathe of the Wild' Zelda game.
+
+## Built With
+
+- *[Create React App](https://github.com/facebook/create-react-app)*
+
+- *Linters for HTML/CSS/JS*
+
+- *GIT/Github*
+
+- *Visual Studio Code*
+
+## Live Demo (if available)
+
+ [Live Version](https://hyrule-compendium-hpz.netlify.app/)
+
+## Getting Started
+
+**To get a local copy up and running follow these simple example steps**.
+
+### Prerequisites
+
+- You need to have a code editor to be able to run the code. VSCode is most preferred.
+
+### Setup
+To get a local copy up and running follw these simple example steps:
+
+```
+    - Select the link with the code on the https code table in the repo
+    - Open a Terminal and write the command: git clone https://github.com/HansZizold/musicartists
+    - $ cd into the directory and run npm install
+    - $ After the node_module module is installed successfully, run npm start
+    - $ The project will open on the browser
+```
+
+### Install
+
+- Run it in your code editor
+
+## Authors
+
+👤 **Hans P. Zizold**
+
+- GitHub: [@githubhandle](https://github.com/HansZizold)
+- Twitter: [@twitterhandle](https://twitter.com/hanzio27)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hans-paul-zizold-37129037/)
+
+## 🤝 Contributing
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
```

Comment:
Please make sure that the "Acknowledgements" section or similar gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Hi @KDlamini, ok you're right. I forgot this requirement.